code/modules/speech/core.dm

```diff
@@ -0,0 +1,248 @@
+/// This atom's listen module tree. May be null if no input modules are registered.
+/atom/var/datum/listen_module_tree/listen_tree = null
+/// The listen effects that this atom *starts* with. It will not be updated nor used again after initialisation.
+/atom/var/list/start_listen_effects = null
+/// The listen modifiers that this atom *starts* with. It will not be updated nor used again after initialisation.
+/atom/var/list/start_listen_modifiers = null
+/// The listen inputs that this atom *starts* with. It will not be updated nor used again after initialisation.
+/atom/var/list/start_listen_inputs = null
+/// The listen languages that this atom *starts* with. It will not be updated nor used again after initialisation. Note that this is the languages that the atom understands when heard.
+/atom/var/list/start_listen_languages = null
```

Comment:
Allll of these feel like they should be on typeinfo, otherwise this is potentially instantiating a massive amount of completely useless lists? Same for the ones below

Reply:
i could see us only wanting this on individual atoms though?